include-fragment-element.js

```diff
@@ -115,8 +127,8 @@ export default class IncludeFragmentElement extends HTMLElement {
           throw new Error(`Failed to load resource: the server responded with a status of ${response.status}`)
         }
         const ct = response.headers.get('Content-Type')
-        if (!ct || !ct.match(/^text\/html/)) {
-          throw new Error(`Failed to load resource: expected text/html but was ${ct}`)
+        if (this.accept !== '*/*' && (!ct || !ct.match(this.accept ? this.accept : /^text\/html/))) {
```

Comment:
We probably want to handle headers like `Accept: text/html; fragment, */*;q=0.5`.

```js
function isWildcard(accept: string): boolean {
  return !!accept.split(',').find(x => x.match(/^\s*\*\/\*/))
}

if (!isWildcard(this.accept) && (!ct || !ct.match(this.accept ? this.accept : /^text\/html/))) …
```